solidity/contracts/bridge/Bridge.sol

```diff
@@ -762,7 +768,40 @@ contract Bridge is Ownable, EcdsaWalletOwner {
         emit VaultStatusUpdated(vault, isTrusted);
     }
 
-    // TODO: updateDepositParameters
+    /// @notice Updates parameters of deposits.
+    /// @param depositDustThreshold New value of the deposit dust threshold in
+    ///        satoshis. It is the minimal amount that can be requested to
+    ////       deposit. Value of this parameter must take into account the value
+    ///        of `depositTreasuryFeeDivisor` and `depositTxMaxFee` parameters
+    ///        in order to make requests that can incur the treasury and
+    ///        transaction fee and still satisfy the depositor
+    /// @param depositTreasuryFeeDivisor New value of the treasury fee divisor.
+    ///        It is the divisor used to compute the treasury fee taken from
+    ///        each deposit and transferred to the treasury upon sweep proof
+    ///        submission. That fee is computed as follows:
+    ///        `treasuryFee = depositedAmount / depositTreasuryFeeDivisor`
+    ///        For example, if the treasury fee needs to be 2% of each deposit,
+    ///        the `depositTreasuryFeeDivisor` should be set to `50`
+    ///        because `1/50 = 0.02 = 2%`
+    /// @param depositTxMaxFee New value of the deposit tx max fee in satoshis.
+    ///        It is the maximum amount of BTC transaction fee that can
+    ///        be incurred by each swept deposit being part of the given sweep
+    ///        transaction. If the maximum BTC transaction fee is exceeded,
+    ///        such transaction is considered a fraud
+    /// @dev Requirements:
+    ///      - Deposit treasury fee divisor must be greater than zero
```

Comment:
Shouldn't we require `depositDustThreshold > 0`? 0-deposit does not make sense 🤔 Unless this is already taken care of in Bitcoin cc @lukasz-zimnoch

Reply:
Done in the follow-up PR in https://github.com/keep-network/tbtc-v2/pull/229/commits/c0a2ec4316c35a476d000a24e6e5c4d479c4c800

Deposit dust threshold is used for checking the value of deposits:
```
         require(
            fundingOutputAmount >= self.depositDustThreshold,
            "Deposit amount too small"
        );
```
Since there can not be deposits with value `0`, we require it to be at least `1`.